indico/modules/events/papers/templates/display/_paper_list.html

```diff
@@ -0,0 +1,64 @@
+{% from 'events/management/_lists.html' import render_displayed_entries_fragment %}
+
+{% macro render_user_papers_list(event, contributions, to_review=false, reviewed=false) %}
+    <section>
+        {% if to_review %}
+            {% set reviewing_list_id = 'to-review-list' %}
+        {% elif reviewed %}
+            {% set reviewing_list_id = 'reviewed-list' %}
+        {% endif %}
+        <div {{ 'id={}'.format(reviewing_list_id) if reviewing_list_id }} class="paper-contribution-list">
```

Comment:
I'd just call the variable `id` and use this syntax:

    <div {% if id %}id="{{ id }}"{% endif %} class="...">